packages/shared-ux/error_boundary/src/services/error_boundary_services.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { FC, useContext } from 'react';
+
+import { ErrorBoundaryServices } from '../../types';
+import { ErrorService } from './error_service';
+
+const Context = React.createContext<ErrorBoundaryServices | null>(null);
+
+/**
+ * A Context Provider for Jest and Storybooks
+ */
+export const ErrorBoundaryProvider: FC<ErrorBoundaryServices> = ({
+  children,
+  reloadWindow,
+  errorService,
+}) => {
+  return <Context.Provider value={{ reloadWindow, errorService }}>{children}</Context.Provider>;
+};
+
+/**
+ * Kibana-specific Provider that maps dependencies to services.
+ */
+export const ErrorBoundaryKibanaProvider: FC = ({ children }) => {
+  const reloadWindow = () => window.location.reload();
```

Comment:
Not sure about this, it feels like just a bit of indirection.

In the future, I also plan to add a `ToastsService` to the mix of services. This will maintain the internal state about the toast objects that get sent to the `EuiGlobalToastList`.